03-other-topics/008-zero-downtime-deployment.md

```diff
@@ -0,0 +1,33 @@
+---
+category: cloud-platform
+expires: 2019-08-12
+---
+
+# Zero Downtime Deployments
+
+## Introduction
+
+In the past at the MoJ, application team members were used to application service outage when deploying a new version of their application.
+
+With the Cloud Platform, this is now an annoyance of the past, thanks to Rolling Updates.
+
+## Rolling Updates
+
+Rolling updates introduce the ability to update an application without any downtime.
+
+A rolling update works by ensuring there is always one extra Pod than the maximum number stated in the deployment.
+
+The new deployment is applied incrementally, one Pod at a time until all Pods are running the latest version.
```

Comment:
This is a good example but in reality the actual numbers are configurable (ie. you can get it to say replace two pods at a time). Can we provide a link to the docs for how to configure the rolling update strategy?